src/planner/operator/schema.cpp

```diff
@@ -1,7 +1,10 @@
 #include "planner/logical_plan/logical_operator/schema.h"
 
+#include "binder/expression/expression_visitor.h"
 #include "common/exception.h"
 
+using namespace kuzu::binder;
```

Comment:
what is our convension of using `using namespace`?

Reply:
We are free to use namespace in cpp files